adservertargeting/reqdataprocessor.go

```diff
@@ -0,0 +1,104 @@
+package adservertargeting
```

Comment:
The term "processor" is not descriptive of the functionality IMHO. How about something like "requestlookup"? .. or just a description which better translates to "looks up and returns a value".

Reply:
Renamed